protocols/relay/examples/relay.rs

```diff
@@ -24,35 +24,53 @@
 //! relay client listening via the relay server and (3) a dialing relay client
 //! dialing the listening relay client via the relay server.
 //!
-//! 1. To start the relay server, run `cargo run --example=relay --package=libp2p-relay --mode relay --secret-key-seed 1 --address /ip4/<ip address>/tcp/<port>`.
-//!    The `-secret-key-seed` helps create a static peer id using the given number argument as  a seed.
-//!    The mode specifies whether the node should run as a relay server, a listening client or a dialing client.
-//!    The address specifies a static address. Usually it will be some loop back address such as `/ip4/0.0.0.0/tcp/4444`.
-//!    Example:
-//!    `cargo run --example=relay --package=libp2p-relay -- --mode relay --secret-key-seed 1 --address /ip4/0.0.0.0/tcp/4444`
-//!    `cargo run --example=relay --package=libp2p-relay -- --mode relay --secret-key-seed 1 --address /ip6/::/tcp/4444`
+//! 1. To start the relay server, run
+//! ```bash
```

Comment:
If I am not mistaken, this would not render correctly in markdown. While not relevant in this case, I wonder whether it would in future cases?

Reply:
I don't understand what what would be incorrect about the rendering? This is what it looks like if I let CLion render it:

![image](https://user-images.githubusercontent.com/5486389/132288571-999b6516-3f2b-459e-a7b6-2202915ab1f1.png)

Reply:
Sorry to be a bit nit picky. The paragraph starting with "The `-secret-key-seed` helps create [...]" should be indented on the level of "1.", but in the rendered version above it is not indented at all.